src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="big"
+      >
+        <div class="icon-big">
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try &#34;Los Angeles&#34;"
+            class="search__input big"
+          />
+        </div>
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
```

Comment:
Add an empty line before this multiline sibling block to improve readability, according to the checklist.